examples/eks/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    castai     = {
+      source  = "castai/castai"
+      version = "0.0.0-local"
```

Comment:
remove the explicit version from the example (we use it only in localdev example) as it isn't going to work for theprovider users. Users will download latest public version from registry.